django_basecrm/helpers.py

```diff
@@ -190,15 +203,18 @@ def get_users():
         app_conf.instantiate_users()
         users = app_conf.users
     else:
-        users = get_users_from_api()
+        users = get_users_from_api(**kwargs)
```

Comment:
This is the second half of the change - where the kwargs are passed to `get_users_from_api`.